app/controllers/api/v0/pages_controller.rb

```diff
@@ -53,9 +46,16 @@ def index
   end
 
   def show
+    # NOTE: This check can be removed once this issue is resolved.
+    # https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/274
+    if should_include_versions
+      raise Api::NotImplementedError, 'The ?include_versions query argument has been disabled temporarily.'
+    end
     page = Page.find(params[:id])
     data = page.as_json(include: [:maintainers, :tags])
-    data['versions'] = page.versions.where(different: true).as_json
+    # NOTE: We'll re-attach versions to /pages/{page_id} when this issue is resolved.
+    # https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/274
+    # data['versions'] = page.versions.where(different: true).as_json
```

Comment:
Same issue here about keeping the functionality and gating it based on configuration.